daemon/container.c

```diff
@@ -338,8 +338,10 @@ container_new_internal(const uuid_t *uuid, const char *name, container_type_t ty
 
 	// network interfaces from container config
 	for (list_t *elem = net_ifaces; elem != NULL; elem = elem->next) {
-		DEBUG("List element in net_ifaces: %s", (char *)(elem->data));
-		nw_mv_name_list = list_append(nw_mv_name_list, mem_strdup(elem->data));
+		char *if_name = elem->data;
+		DEBUG("List element in net_ifaces: %s", if_name);
+		if (cmld_netif_phys_remove_by_name(if_name))
```

Comment:
handle case that if is not present at start but added later? (possibly with a blacklist)

Reply:
yeah can be another feature commit later.